include/pybind11/pytypes.h

```diff
@@ -2183,6 +2183,11 @@ class list : public object {
             throw error_already_set();
         }
     }
+    void clear() {
```

Comment:
Can you add the non-const comment the other pybind11 clear methods have?

Reply:
Done. Thanks for catching this!